src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution.cs

```diff
@@ -3025,6 +3026,7 @@ private RefKind GetEffectiveParameterRefKind(ParameterSymbol parameter, RefKind
             TypeSymbol parameterType,
             RefKind parRefKind,
             bool ignoreOpenTypes,
+            bool isThisArgument,
```

Comment:
I can simply pass `argRefKind = RefKind.Ref` for ref extension methods, but will have to complete the tests for supported conversion kinds to confirm.

---
In reply to: [120724731](https://github.com/dotnet/roslyn/pull/19703#discussion_r120724731) [](ancestors = 120724731)